compact.go

```diff
@@ -593,9 +616,20 @@ func (c *LeveledCompactor) populateBlock(blocks []BlockReader, meta *BlockMeta,
 		allSymbols = make(map[string]struct{}, 1<<16)
 		closers    = []io.Closer{}
 	)
-	defer func() { closeAll(closers...) }()
+	defer func() {
+		closeAll(closers...)
```

Comment:
I think we should not ignore the error here. Either log or return by binding it to the return variables.

Reply:
updated, thanks!